okhttp-dnsoverhttps/src/test/java/okhttp3/dnsoverhttps/DnsOverHttpsTest.java

```diff
@@ -84,6 +84,18 @@
         "/lookup?ct&dns=AAABAAABAAAAAAAABmdvb2dsZQNjb20AAAEAAQ");
   }
 
+  @Test public void getIPAddressHost() throws Exception {
+    dns = buildLocalhost(bootstrapClient, true);
+
+    String hostIPv4 = "127.0.0.1";
+    List<InetAddress> resultIPv4 = dns.lookup(hostIPv4);
+    assertThat(resultIPv4).contains(address(hostIPv4));
+
+    String hostIPv6 = "2a03:2880:f029:11:face:b00c:0:2";
```

Comment:
Can you add tests for [::1] and ::1

Reply:
Sure, just addes test cases for these addresses.